languages/dart/keywords/Function.md

```diff
@@ -0,0 +1,33 @@
+# `function`
```

Comment:
nit: The filename should be `function.md` to be consistent with other files. 

The `Function` is a built-in type, where as `function` is a keyword. Perhaps we should mention that somewhere.

Reply:
Renamed, the keywords can be expanded upon anyone that wants to work on this.